dnsimple/response.py

```diff
@@ -36,11 +36,14 @@ def __init__(self, http_response, obj=None):
             When the server responds with an error code
         """
         if int(http_response.status_code) in range(400, 504):
-            message = http_response.json().get('message')
-            errors = None
-            if http_response.json().get('errors'):
-                errors = http_response.json().get('errors')
-            raise DNSimpleException(message=message, errors=errors)
+            if http_response.text != '':
+                message = http_response.json().get('message')
+                attribute_errors = None
+                if http_response.json().get('errors'):
+                    attribute_errors = http_response.json().get('errors')
```

Comment:
```suggestion
                attribute_errors = http_response.json().get('errors')
```

IMHO is not needed to parse the json twice, if the .json().get('errors') does not exist it will return None